tests/test_metadata.py

```diff
@@ -0,0 +1,25 @@
+import marshmallow_recipe as mr
+
+
+def test_metadata_1() -> None:
+    metadata1 = mr.Metadata(dict(a=1, b=2))
+    metadata2 = mr.Metadata(dict(b=2, a=1))
+
+    assert hash(metadata1) == hash(metadata2)
+    assert metadata1 == metadata2
+
+
+def test_metadata_2() -> None:
+    metadata1 = mr.Metadata(dict(a=1, b=1))
+    metadata2 = mr.Metadata(dict(b=2, a=2))
+
+    assert hash(metadata1) == hash(metadata2)
+    assert metadata1 != metadata2
+
+
+def test_metadata_3() -> None:
```

Comment:
- test_metadata_1 -> test_metadata_different_keys_order
- test_metadata_2 -> test_metadata_not_equal_same_hash
- test_metadata_3 -> test_metadata_not_equal

or something like this but not 1/2/3

Reply:
Thanks